### DISCUSSION TITLE
連防毒軟體公司都被駭！專家警告：不要再用 Chrome 記住密碼了

### DISCUSSION THREAD

**Discussion by blackline0911** [1101-GS4719-C/Python-Community-GS4719-C#626] [General]
https://buzzorange.com/techorange/2022/01/04/chrome-edge-password/

許多人習慣用 Chrome 和 Microsoft Edge 記住密碼，不必每次登入都得重新輸入，一般也認為很安全，但專家表示，在家工作的民眾如果把密碼儲存在網路瀏覽器，很容易成為駭客目標。

惡意軟體 Redline Stealer 太強，連防毒軟體都沒輒
根據防毒軟體公司 Ahnlab，有一名遠端上班的員工利用 VPN 帳戶登入公司內部網路，結果受駭。

這個人在與其他人共享的裝置上工作，全然不知該裝置已經感染了竊取訊息的惡意軟體 Redline Stealer。

各位...注意資安安全啊

**Comment by awkward-willy** [comment]
記住密碼雖然很方便不過還是有很大的疑慮呢
可見密碼還是得要自己用腦袋記比較安全一點
同時還要記得定期更換還有使用高強度密碼
有相當多要注意的點呢

**Comment by HsuChen88** [comment]
以後遠端工作也需要注意資安的問題

**Comment by henrywen547** [comment]
記住密碼真的功能真的蠻方便的
不過確實是因為資安考量
現在在chrome上記住的密碼越來越少
現在都留在記事本中

**Comment by Rosancl** [comment]
以前貪方便會用記住密碼，後來也因為資安問題都沒再用了

**Comment by Jay-722** [comment]
蛤阿，記住密碼真的很方便ㄟ

**Comment by Yao900108** [comment]
我也有使用這個功能.....看來之後要小心點了

**Comment by 110409538** [comment]
我就是其中一個會讓系統幫忙記密碼的人。雖然有危險性但真的很方便。

**Comment by Ben6uo** [comment]
算了 真正怕有資安問題的話還是推薦不要用電子產品了

**Comment by HaoFish123** [comment]
存了蠻多密碼的，真可怕，不過還是謝謝分享。

**Comment by bloodtrail0511** [comment]
危 我幾乎所有帳號都存在chrome上了餒
